src/pages/home.js

```diff
@@ -0,0 +1,18 @@
+import React from 'react';
+
+//requisito 2
+
+class Home extends React.Component {
+  render () {
+    return (
+      <div>
+        <input className='App' type='text' />
+        <p data-testid='home-initial-message'>
```

Comment:
Correção apontada pelo ESLint
```suggestion
        <p data-testid="home-initial-message">
```